examples/app/src/index.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { ThemeProvider, lightTheme } from "@strapi/design-system";
+import { GlobalStyle } from "@buffetjs/styles";
+import "./index.css";
+import Main from "./App";
+import reportWebVitals from "./reportWebVitals";
+
+const App = () => {
+  return (
+    <ThemeProvider theme={lightTheme}>
+      <GlobalStyle />
```

Comment:
Maybe you can create 2 pages, one for buffet one for the new DS so we can easily see the difference between them?

Reply:
Not sure it is relevant to keep buffet in this app. I will remove it.